wifi-subsys/components/network/src/httpsclient.c

```diff
@@ -0,0 +1,150 @@
+#include <string.h>
+#include <stdlib.h>
+#include "httpsclient.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/event_groups.h"
+#include "esp_wifi.h"
+#include "esp_event.h"
+#include "esp_log.h"
+#include "esp_system.h"
+#include "nvs_flash.h"
+#include "esp_netif.h"
+#include "lwip/err.h"
+#include "lwip/sockets.h"
+#include "lwip/sys.h"
+#include "lwip/netdb.h"
+#include "lwip/dns.h"
+#include "esp_tls.h"
+#include "esp_crt_bundle.h"
+
```

Comment:
Are you sure you need all these libraries?

Reply:
I need the following libraries:
#include "esp_tls.h"
#include "esp_crt_bundle.h"
thanks!.